VideoWeb/VideoWeb/Startup.cs

```diff
@@ -33,8 +33,9 @@ public void ConfigureServices(IServiceCollection services)
             services.AddCustomTypes();
 
             services.RegisterAuthSchemes(Configuration);
-            services.AddMvc(opt => opt.Filters.Add(typeof(LoggingMiddleware))).SetCompatibilityVersion(CompatibilityVersion.Version_3_0);
-            services.AddMvc().SetCompatibilityVersion(CompatibilityVersion.Version_3_0).AddFluentValidation();
+            var compatibilityVersion = CompatibilityVersion.Version_3_0;
```

Comment:
If this is a recommendation from resharper it may be worth having rules added to the repo so different users do not get different recommendations.

Reply:
Removed the SetCompatibilityVersion method as it is a no-op for ASP.NET Core 3.0 apps. Calling SetCompatibilityVersion with any value of CompatibilityVersion has no impact on the application.

See: https://docs.microsoft.com/en-us/aspnet/core/mvc/compatibility-version?view=aspnetcore-3.1